CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read this guide to learn how to get started. More information about the individual sub-repos can be found in the relevant README.md, as well as on the [airswap docs site](https://docs.airswap.io/).
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
```

Comment:
Not sure how helpful this example is as it doesn't fork the repo and instead just clones the original directly?

Reply:
git clone https://github.com/<YOUR-GITHUB-USER>/airswap-protocols` as an update